DMCompiler/Compiler/DMPreprocessor/DMPreprocessor.cs

```diff
@@ -754,6 +754,13 @@ private List<Token> GetLineOfTokens()
                         parameters.Add(currentParameter);
                         currentParameter = new List<Token>();
                         parameterToken = GetNextToken(true);
+                        while(parameterToken.Type == TokenType.Newline || parameterToken.Type == TokenType.DM_Whitespace) {
```

Comment:
parameterToken is already gotten with `GetNextToken(true)` which should ignore whitespace tokens, making the check for `TokenType.DM_Whitespace` here unneeded.